### ISSUE TITLE
Error 500 Internal Server Error

### ISSUE THREAD

**Issue by janecker** [Homegear/Homegear#357]
Hi,

first of all thank you for developing homegear! I am using it for quite some time now and it is very essential for my smarthome.

I am running homematic as a Docker container and I am facing internal server errors when I try to access various admin pages. I suspect the reason is a mistake I made: I had the container running on the nightly image (for some reason I switched to the nightly at some point, but don't remember why anymore). Later I added a watchtower container, and at some random day my homegear stopped working becuase of some automatic update watchtower did.  I got it working somehow by revering the stable image but it seems my installation is somehow broken since than.

This happend quite some weeks ago and homegear is mostly working (all devices are working as expected) but I cannot access multiple things in the admin UI, for example the device configuration page. I only get a "500 Internal Server Error".

Following some warnings and errors I found in the log files.

homegear-scriptengine.err:
```
01/20/22 17:49:36.059 Script engine (/var/lib/homegear/www/rpc/ui/signin.php): PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in /var/lib/homegear/ui/signin.php on line 67
sh: 1: /sbin/ifconfig: not found
2022-01-20T17:50:34+01:00 [critical] Uncaught PHP Exception RuntimeException: "Controller "App\Controller\Admin\Inventory\DeviceController::editAction()" requires that you provide a value for the "$utils" argument. Either the argument is nullable and no null value has been provided, no default value has been provided or because there is a non optional argument after this one." at /var/lib/homegear/admin-ui/vendor/symfony/http-kernel/Controller/ArgumentResolver.php line 80
01/20/22 17:50:36.119 Script engine (/var/lib/homegear/www/rpc/ui/signin.php): PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in /var/lib/homegear/ui/signin.php on line 67
```

homegear-scriptengine.log:
```
01/20/22 17:51:36.175 Info: Script 21758 exited with code 0.
01/20/22 17:52:36.200 Info: Script 21759 exited with code 0.
01/20/22 17:52:36.213 Info: Script 21760 exited with code 0.
01/20/22 17:52:36.220 Script engine (/var/lib/homegear/www/rpc/ui/signin.php): PHP Warning:  Undefined array key "HTTP_ACCEPT_LANGUAGE" in /var/lib/homegear/ui/signin.php on line 67
01/20/22 17:52:36.222 Info: Script 21761 exited with code 0.
```

homegear-management.err:
```
01/10/22 00:04:13.993 Connection to IPC server closed (2).
/bin/sh: 1: systemctl: not found
/bin/sh: 1: systemctl: not found
/bin/sh: 1: systemctl: not found
/bin/sh: 1: systemctl: not found
```

homegear-webssh.err:
```
01/10/22 00:04:13.993 Connection to IPC server closed (2).
01/10/22 00:04:46.852 Warning: Can't invoke method ptyOutput as there is no open IPC connection.
01/10/22 00:04:46.852 Error: Could not call ptyOutput: Unknown application error.
01/10/22 00:04:46.952 Warning: Can't invoke method ptyOutput as there is no open IPC connection.
01/10/22 00:04:46.952 Error: Could not call ptyOutput: Unknown application error.
```

Is there any way to repair my setup?

**Comment by m-scheffler** [comment]
Hi @janecker,

unfortunately I'm not able to reproduce the error when trying to edit a device. I'm using:
```
UI-Version  0.2.0-1321-dev
Version  0.8.22012307-3490
```

The 500 when opening the System Settings is fixed and will be available in the nightly soon.

-- Micha

**Comment by janecker** [comment]
I just got around to spend some time on the issue again. I created a new docker container and afterwards compared all my files from etc and lib to the ones created by the new container. I carefully synced the files and merged all the configs. There where quite some changes in variable names / values in the configs (especaly rpcservers.conf). I tried to restore as many default settings as possible and only kept stuff where I had the feeling they are required modifications for my setup. I feard I would have to do something to the sqlite database, but lukily this was not the case. After the changes I am now finally able to access my device configurations again.


There is only one small difference that I could spot on the Settings -> System page. On my old install the page is completly empty (except the heading). On the fresh install I get the message that I am not allowed to read the settings.

Old install:
![image](https://user-images.githubusercontent.com/1313022/156880136-4ef19555-cd59-4645-9a08-64802cf2dc7f.png)

New install:
![image](https://user-images.githubusercontent.com/1313022/156880125-ac7886df-73ee-43d8-8fa9-4ca956d860af.png)
